maestro-android/src/androidTest/java/dev/mobile/maestro/ViewHierarchy.kt

```diff
@@ -93,6 +93,7 @@ object ViewHierarchy {
             serializer.attribute("", "NAF", java.lang.Boolean.toString(true))
         }
         serializer.attribute("", "index", Integer.toString(index))
+        serializer.attribute("", "hintText", safeCharSeqToString(node.hintText))
```

Comment:
Do we need one of these for `accessibilityText` as well?

Reply:
So, accessibilityText is a platform agnostic added new attribute. The one attribute which resolves to this is `content-desc` (contentDescription) which is already here.

Reply:
Got it, thanks!